system/applications/global.nix

```diff
@@ -244,6 +244,8 @@ in {
         ping = "gping"; # ping with a graph
         reboot-windows =
           "sudo efibootmgr --bootnext ${config.boot.windowsEntry} && reboot"; # Reboot to windows
+        repair-store =
+          "nix-store --verify --check-contents --repair"; # Verifies integrity and repairs internal nix inconsistencies
```

Comment:
The comment should point out that it repairs store inconsistencies, not nix ones.

Reply:
According to the nix manual, it verifies the internal consistency of the Nix database, and the consistency between the Nix database and the Nix store, not the Nix store itself.

Reply:
So maybe "Verifies integrity and repairs inconsistencies between Nix database and store" would be the best.